src/core/src/command/transfer.rs

```diff
@@ -75,9 +75,7 @@ pub struct BufferImageCopy {
     ///
     pub buffer_slice_pitch: u32,
     ///
-    pub image_aspect: image::AspectFlags,
-    ///
-    pub image_subresource: image::SubresourceLayers,
+    pub image_range: image::SubresourceRange,
```

Comment:
I'm not really sure about the general removal of `SubresourceLayers` as we often end up with ranges where we only use the `start` of the mip levels. Using `SubrsourceRanges` suggests that we could for example copy multiple mipmap layers here which is wrong, resulting in a run-time error.

Reply:
Resolved now